app/src/test/kotlin/net/rafaeltoledo/social/MainActivityTest.kt

```diff
@@ -1,16 +1,29 @@
 package net.rafaeltoledo.social
 
-import org.junit.Assert.assertNotNull
+import android.widget.TextView
+import com.google.common.truth.Truth.assertThat
 import org.junit.Test
 import org.junit.runner.RunWith
 import org.robolectric.Robolectric
 import org.robolectric.RobolectricTestRunner
+import org.robolectric.RuntimeEnvironment
 
 @RunWith(RobolectricTestRunner::class)
 class MainActivityTest {
 
     @Test
     fun checkIfActivityIsSuccessfullyCreated() {
-        assertNotNull(Robolectric.setupActivity(MainActivity::class.java))
+        // Arrange
+        val newValue = "Test Social App"
+        val app = RuntimeEnvironment.application as TestSocialApp
+        app.initKoin(newValue)
+
+        // Act
+        val activity = Robolectric.setupActivity(MainActivity::class.java)
```

Comment:
This seems more like an arrange as well... I know that in this case we would end up with an empty Act comment, but you are not actually doing nothing in the Activity so it should be explicit that there is no action for the assertions below.